File: python/7 kyu/205.py
# https://www.codewars.com/kata/5ae7e3f068e6445bc8000046/train/python

# Passed

def next_happy_year(year):
    year += 1
    while len(str(year)) != len(set(str(year))):
        year += 1
        
    return year

output = next_happy_year(1987)
print(output)

File: nodejs/8 kyu/205.js
// https://www.codewars.com/kata/515e271a311df0350d00000f/train/javascript

// Passed

function squareSum(numbers) {
  let numbersSquared = numbers.map((num) => num ** 2);
  let sumOfNumbersSquared = numbersSquared.reduce(
    (prev, curr) => prev + curr,
    0
  );
  return sumOfNumbersSquared;
}

const output = squareSum([1, 2]);
console.log(output);